packages/dom/src/feature-app-container.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  FeatureAppDefinition,
+  FeatureAppManager,
+  FeatureAppScope
+} from '@feature-hub/core';
+import {LitElement, html, property} from 'lit-element';
+import {TemplateResult} from 'lit-html';
+
+/**
+ * A DOM Feature App allows the use of any frontend technology such as Vue.js,
+ * React, or Angular.
+ */
+export interface DomFeatureApp {
+  /**
+   * @param container The container element to which the Feature App can attach
+   * itself.
+   */
+  attachTo(container: Element): void;
+}
+
+export function createFeatureAppContainer(
+  manager: FeatureAppManager
```

Comment:
I thought we were currently writing the name out everywhere. Isn't that so? `featureAppManager`